src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -139,7 +139,10 @@ public void exit(int status) {
      * them run concurrently.  When all the hooks have finished it will then
      * halt. Note that daemon threads will continue to run during the shutdown
      * sequence, as will non-daemon threads if shutdown was initiated by
-     * invoking the {@link #exit exit} method.
+     * invoking the {@link #exit exit} method. Also note that running threads
+     * can create and start new threads during the shutdown sequence; the new
+     * threads execute concurrently with their creators and may create and
+     * start further threads.
```

Comment:
True - but does it really need to be singled out for mention? I think explicitly pointing this out will just lead to follow up queries as to how such threads interact with the shutdown sequence.